tests/e2e/cucumber/steps/app-files/page/spaces/projects.ts

```diff
@@ -90,6 +103,17 @@ Then(
   }
 )
 
+Then(
+  '{string} should not be able to see space {string}',
+  async function (this: World, stepUser: string, space: string): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const spacesObject = new objects.applicationFiles.Spaces({ page })
+    const spaceID = spacesObject.getSpaceID({ key: space })
+    const isVisible = await spacesObject.spacesIdExist(spaceID)
+    expect(isVisible).toBe(true)
```

Comment:
looks illogical. If user shouldn't see space i expect: is space visible? -> not(false)

Reply:
@ScharfViktor are you suggesting to use other logic rather than a boolean assertion in here?

Reply:
If I'm not wrong I think Viktor meant since the step is `should not be able to see space` the logic 
```
const isVisible = await spacesObject.spacesIdExist(spaceID)
   expect(isVisible).toBe(true)
```
seems wrong. many be it should be something like
```
const isNotVisible = await spacesObject.spacesIdExist(spaceID)
   expect(isNotVisible).toBe(true)
```
OR

```
const isVisible = await spacesObject.spacesIdExist(spaceID)
   expect(isVisible).toBe(false)
```

Reply:
I'm sorry I'm late. @SwikritiT wrote what I meant.